wpilibc/src/main/native/include/frc/smartdashboard/Mechanism2d.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include <wpi/StringMap.h>
+#include <wpi/mutex.h>
+
+#include "networktables/NetworkTableEntry.h"
+#include "frc/smartdashboard/Sendable.h"
+#include "frc/smartdashboard/SendableHelper.h"
+#include "frc/util/Color8Bit.h"
+
+#include "MechanismRoot2d.h"
+
+namespace frc {
+
+class Mechanism2d : public Sendable, public SendableHelper<Mechanism2d> {
```

Comment:
Add a documentation comment for each top-level class (applies to Java too) with a description of the class.